libs/application/templates/health-insurance/src/fields/Review/StatusAndChildren.tsx

```diff
@@ -113,9 +104,10 @@ const StatusAndChildren: FC<ReviewFieldProps> = ({
                   formatMessage,
                 )}
               />
-              <InputFileUpload
+              <FileUploadController
+                application={application}
                 id="confirmationOfStudies"
-                disabled={!isEditable}
+                maxSize={10000000}
```

Comment:
As we use this value in so many places, we should create a constant for it so that any changes updates everywehere it is used